### PR TITLE
feat(home): updating home community id

### PR DESCRIPTION
**Pull request by jonahkaye**
Refs: #[1381](https://github.com/metriport/metriport-internal/issues/1381)

### Description

- updating the home Community Id. 
- fixing messageId which is undefined
- qvera wants it

### Release Plan

- [ ] Merge this asap to develop
- [ ] safe to go to production. no prod changes. no infra updates.

### CODE REVIEW

File: packages/api/src/external/carequality/mock-patient-discovery.ts [hunk 1]
```diff
@@ -35,10 +35,10 @@ export async function mockResult(id: string, cxId: string, patientId: string) {
     responseTimestamp: "2023-11-29T08:41:47.047-0800",
     patientId,
     patientMatch: true,
-    xcpdHomeCommunityId: "1.2.840.114350.1.13.11511.3.7.3.688884.100.1000",
+    xcpdHomeCommunityId: "2.16.840.1.113883.3.9621",
```
Comment (RamilGaripov) [inline packages/api/src/external/carequality/mock-patient-discovery.ts:38]: This attribute should be called `gatewayHomeCommunityId` now. The schema was updated in [this PR](https://github.com/metriport/metriport/pull/1331), but looks like I forgot to update this mock template. Let's assign the type to this constant on initialization to enable type checking, please.

File: packages/api/src/external/carequality/mock-patient-discovery.ts [hunk 2]
```diff
@@ -35,10 +35,10 @@ export async function mockResult(id: string, cxId: string, patientId: string) {
...
     xcpdPatientId: {
```
Comment (RamilGaripov) [inline packages/api/src/external/carequality/mock-patient-discovery.ts:39]: And this needs to be `gatewayPatientId`

**Review by RamilGaripov — APPROVED** [review]
Approved on the condition these comments are addressed

File: packages/api/src/external/carequality/mock-patient-discovery.ts (hunk 1 above)
Comment (leite08) [reply packages/api/src/external/carequality/mock-patient-discovery.ts:38]: > assign the type to this constant on initialization to enable type checking

image

**Pull request merged**